v3/internal/limits.go

```diff
@@ -13,10 +13,9 @@ const (
 	// MaxPayloadSizeInBytes specifies the maximum payload size in bytes that
 	// should be sent to any endpoint
 	MaxPayloadSizeInBytes = 1000 * 1000
-
 	// MaxCustomEvents is the maximum number of Transaction Events that can be captured
 	// per 60-second harvest cycle
-	MaxCustomEvents = 10 * 1000
+	MaxCustomEvents = 30 * 1000
```

Comment:
Are we ready to increase the max number of custom events to 30,000 right now?

Reply:
We could hold it until performance testing is done, or we can keep it in. Regardless, the max custom events are still going to be server limited to 10,000.

Reply:
I think that we should keep defaults the same as they historically have been, just in case things change on the server side. We want to be very explicitly in control of when we expose customers to those kinds of changes.

Reply:
Got it. I will revert that change back.